bpf/lib/overloadable_xdp.h

```diff
@@ -152,7 +167,7 @@ ctx_set_encap_info(struct xdp_md *ctx __maybe_unused,
 		   __u32 node_id __maybe_unused,
 		   __u32 seclabel __maybe_unused,
 		   __u32 dstid __maybe_unused,
-		   __u32 vni __maybe_unused, __u32 *ifindex __maybe_unused)
+		   __u32 vni __maybe_unused, int *ifindex __maybe_unused)
```

Comment:
Heh, I actually wanted to switch all the remaining ifindex usage to `__u32`. Some other time then.